src/screens/General/About.js

```diff
@@ -1,76 +1,169 @@
 import React, { Component } from "react";
-import { View, SafeAreaView, FlatList, Dimensions } from "react-native";
+import { Avatar } from "react-native-elements";
 import { Actions } from "react-native-router-flux";
 import { ListItem } from "react-native-elements";
 import { NavBar } from "../../components/general";
 import Ionicons from "react-native-vector-icons/Ionicons";
+import { View, Text, SafeAreaView, FlatList, Dimensions, Linking, TouchableOpacity } from "react-native";
+import list from "../../data/AboutItems.json";
 
 const dimension = Dimensions.get("window");
-const menuItems = [
-	// {
-	// 	title: "E-Board",
-	// 	icon: "people",
-	// 	screen: "EBoard"
-	// },
-	{
-		title: "Privacy Policy",
-		icon: "insert-drive-file",
-		screen: "WebPageShow",
-		uri: "http://www.shpeucf.com/privacy-policy/"
-	},
-	{
-		title: "Version",
-		icon: "beenhere",
-		screen: "Version",
-		content: "Current version: v1.1",
-		footer: "Check our GitHub Page - SHPEUCF"
-	}
-];
 
 class About extends Component {
 	render() {
+		const {
+			textColor,
+			bgColor,
+			center,
+			titleStyle,
+			contributorStyle,
+			containerFlex,
+			footer
+		} = styles;
+
 		return (
-			<SafeAreaView style = {{ backgroundColor: "#0c0b0b", flex: 1 }}>
-				<NavBar title = "About" back onBack = { () => Actions.pop() } ></NavBar>
-				<View style = {{ backgroundColor: "black" }}>
+			<SafeAreaView style = { [{ backgroundColor: "#0c0b0b" }, containerFlex] }>
```

Comment:
Why did you replace { backgroundColor: "black" } with a prop but not { backgroundColor: "#0c0b0b" }

Reply:
The "black" background color is used multiple types, while #0c0b0b is used once. Should I add it as another style for consistency?